src/components/confirm/confirm.d.ts

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+
+export interface ConfirmProps {
+  onConfirm: () => void;
+  // ** Customise the confirm button label */
+  confirmLabel?: string;
+  // ** Customise the cancel button label */
+  cancelLabel?: string;
+  /** Apply destructive style to the buttons */
+  destructive?: boolean;
+  /** Defines an Icon type within the button (see Icon for options) */
+  iconType?: "error" | "warning";
+  /** Makes cancel button disabled */
+  disableCancel?: boolean;
+  /** Makes confirm button disabled */
+  disableConfirm?: boolean;
+  /** Allows to setup buttonType into cancel button */
+  cancelButtonType?: 'primary' | 'secondary' | 'tertiary' | 'dashed' | 'destructive' | 'darkBackground';
```

Comment:
In the propTypes in the component this can only be `primary`, `secondary` or `tertiary` - it needs to be the same here